source/_patterns/02-molecules/vue-widget/demo/vue-html-example.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <form 
```

Comment:
Is this style of breaking apart html attributes standard in .vue files? Is this a style we should keep consistent in writing for .vue files?